### ISSUE TITLE
Non-explanatory error message when uploading media above 2MB

### ISSUE THREAD

**Issue by nehemiagurl** [pixelfed/pixelfed#3596]
When trying to upload a picture above 2MB, I get this very non-explanatory error message:
![signal-2022-07-29-173150](https://user-images.githubusercontent.com/109424320/181791184-b089dd77-7c3b-4730-af46-442eeb147d52.jpeg)
(When using an alternative client, e.g. PixelDroid, it gets a generic "500: Internal server error" message, which is also non-explanatory.)

In fact, I only realised the threshold was 2MB through trial and error. The error message alone provides no indication this is a size issue.
Is this limit a result of the Pixelfed code? Of the server settings? Of my user settings? I have no idea - which is part of the problem with such a non-explanatory message.

The best solution to deal with too large images, in my opinion, will be to pop up a prompt asking the user whether they would prefer to remove them or to automatically rescale them so they fit, but as I understand this may be harder to implement and a long-term solution, the error message should be changed to something in this vain:

> **Error: image too large**: The image you uploaded has the size of **xxMB**. Unfortunately, only images up to **yyMB** are supported. Please resize the picture before trying again.

**Comment by mbliznikova** [comment]
Hi, I've created a PR addressing the issue, would really appreciate any feedback/review/comments : https://github.com/pixelfed/pixelfed/pull/4684
